packages/terra-form-select/src/terra-dev-site/doc/example/native-select-field/DefaultField.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import NativeSelectField from 'terra-form-select/lib/native-select/NativeSelectField';
+import styles from '../FormSelectDocCommon.module.scss';
+
+const cx = classNames.bind(styles);
+
+const DefaultFieldExample = () => (
+  <NativeSelectField
```

Comment:
Should this example have an aria label?

Reply:
Actually i assume this is handled by being a field and passing in the label.

Reply:
Yes the label serves double duty as the aria label as well. I believe it matches out existing field patterns.